src/classrooms/classrooms.service.ts

```diff
@@ -39,4 +43,21 @@ export class ClassroomsService {
 
     return classroom;
   }
+
+  async addStudent(enrollStudentDto: EnrollStudentDto) {
+    const { studentId, classroomId } = enrollStudentDto;
+
+    const student = await this.studentsRepository.findOneOrFail({ id: studentId });
+
+    const classroom = await this.classroomsRepository.findOneOrFail({ id: classroomId });
+
+    const classroomStudent = this.classroomsStudentsRepository.create({
+      studentId: student,
+      classroomId: classroom,
+    });
```

Comment:
```suggestion
    await this.studentsRepository.findOneOrFail({ id: studentId });

    await this.classroomsRepository.findOneOrFail({ id: classroomId });

    const classroomStudent = this.classroomsStudentsRepository.create({
      studentId,
      classroomId,
    });
```

[NIT] - You don't need the vars. You can just pass the id